go.mod

```diff
@@ -22,8 +16,8 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.2
-	github.com/vmware-tanzu/nsx-operator/pkg/apis v0.0.0-20231227030711-34390278f3cd
-	github.com/vmware-tanzu/nsx-operator/pkg/client v0.0.0-00010101000000-000000000000
+	github.com/vmware-tanzu/nsx-operator/pkg/apis v0.0.0-20240305035435-c992c623aad3
```

Comment:
It's good that we clearly say this uid is in accordance with which exact commit id of pkg/apis in the PR commit msg, it will help us to track this lib change history and what exact commit we provide to users.

Reply:
c992c623aad3 is commit id of pkg/apis lib.

Reply:
Synced offline.